indico/modules/events/contributions/controllers/management.py

```diff
@@ -28,6 +28,7 @@
 from indico.modules.events.abstracts.forms import AbstractContentSettingsForm
 from indico.modules.events.abstracts.settings import abstracts_settings
 from indico.modules.events.contributions import get_contrib_field_types
+from indico.modules.events.contributions.controllers.base import ContributionListMixin
```

Comment:
I don't understand. The `abstracts/controllers/base.py` is full of mixins, including the `DisplayAbstractListMixin` which has essentially the same functionality as the `ContributionListMixin` in `contributions/controllers/base.py`.